### DISCUSSION TITLE
Template for a whole Event

### DISCUSSION THREAD

**Discussion by loelismurf** [liedekef/events-made-easy#203] [Ideas]
I propose a function for the extension of the template function.
Currently a template can be created for each RSVP format and mail, but each time each template must be selected.
This is not possible via parameterizing the templates that have been created.
The idea is that a template can be created with the entire configuration of an event, for example as XML or JSON.
Where the settings can be linked as a template
The same settings as under Events Made Easy -> Settings -> RSVP as Example Booking recorded message or Booking form format (Here a template or a fixed text can be deposited

**Comment by liedekef** [comment]
Asking people to create XML or JSON files is not user friendly in the wordpress world. I even dislike xml in the context of tomcat, shibboleth etc ... 
Currently you can already create an event as draft (so it won't show up in lists/calendar, put it like 5 years in the future (so it is last in the events overview) and configure it to your liking. Then you can use that event as a template for new events by just clicking on the copy-icon in the events overview et voila.
Feel free to contribute code though :-)

**Reply by loelismurf** [reply]
Thanks for the answer, the goal is that my other leader can easly choise but parameterizing.
It is a way to go, may be i will try to code it xD thanks
